app/code/Magento/ReviewGraphQl/README.md

```diff
@@ -0,0 +1,2 @@
+# ReviewGraphQl
+
```

Comment:
Please, do not forget to put some warm words into the README 😎

Reply:
Added 🌞